connectors/src/lib/models/google_drive.ts

```diff
@@ -335,6 +336,10 @@ GoogleDriveWebhook.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    driveId: {
+      type: DataTypes.STRING,
+      allowNull: false,
```

Comment:
How are you going to manage the migration, since existing objects in db don't have the column?

Reply:
Ok thank you for adding the deploy plan in the OP! It was not clear that you would empty the table before running this migration!